Samples/WMCConsole_winrtcpp/WMCConsole_winrtcpp.cpp

```diff
@@ -0,0 +1,255 @@
+    auto file1 = folder.CreateFileAsync(to_hstring(photoIndex) + L"_1.png", CreationCollisionOption::ReplaceExisting).get();
+    auto file2 = folder.CreateFileAsync(to_hstring(photoIndex) + L"_2.png", CreationCollisionOption::ReplaceExisting).get();
+
+    //Capture and save two photos
+    mediaCapture.CapturePhotoToStorageFileAsync(ImageEncodingProperties::CreatePng(), file1).get();
+    mediaCapture.CapturePhotoToStorageFileAsync(ImageEncodingProperties::CreatePng(), file2).get();
+
+    auto sbList = single_threaded_vector<SoftwareBitmap>();
+    // open photos and decode them into NV12 format for processing
+    // photo1
+    {
+        auto strm = file1.OpenReadAsync().get();
+        auto decoder = BitmapDecoder::CreateAsync(strm).get();
+        auto sb1 = decoder.GetSoftwareBitmapAsync().get();
+        sbList.Append(SoftwareBitmap::Convert(sb1, BitmapPixelFormat::Nv12));
+    }
+    // photo2
+    {
+        auto strm = file2.OpenReadAsync().get();
+        auto decoder = BitmapDecoder::CreateAsync(strm).get();
+        auto sb1 = decoder.GetSoftwareBitmapAsync().get();
+        sbList.Append(SoftwareBitmap::Convert(sb1, BitmapPixelFormat::Nv12));
+    }
+
+    // Do some processing like low light fusion in this example
+    auto op = LowLightFusion::FuseAsync(sbList).get();
+
+    // Save the output
+    auto fname = to_hstring(photoIndex) + L"_LLF.png";
+    auto fileOp = folder.CreateFileAsync(fname, CreationCollisionOption::ReplaceExisting).get();
+    {
+        auto strm = fileOp.OpenAsync(FileAccessMode::ReadWrite).get();
+        auto encoder = BitmapEncoder::CreateAsync(BitmapEncoder::PngEncoderId(), strm).get();
+        encoder.SetSoftwareBitmap(SoftwareBitmap::Convert(op.Frame(), BitmapPixelFormat::Bgra8));
+        encoder.FlushAsync().get();
+    }
+}
+
+MediaCapture InitCamera()
+{
+    MediaCapture mediaCapture;
+    auto settings = MediaCaptureInitializationSettings();
+    auto filteredGroups = GetFilteredSourceGroupList();
+
+    if (filteredGroups.Size() < 1)
+    {
+        std::cout << "Error No suitable capture sources found";
+        throw_hresult(MF_E_NO_CAPTURE_DEVICES_AVAILABLE);
+    }
+
+    auto selectedSGidx = GetSGSelection(filteredGroups);
+
+    auto selectedSrc = filteredGroups.GetAt(selectedSGidx);
+
+    if (selectedSrc == nullptr)
+    {
+        throw_hresult(MF_E_OUT_OF_RANGE);
+    }
+
+    settings.SourceGroup(selectedSrc.SourceGroup());
+
+    // if user explicitly selected a non-photo pin to take the photo
+    if (selectedSrc.MediaStreamType() != MediaStreamType::Photo)
+    {
+        settings.PhotoCaptureSource(PhotoCaptureSource::VideoPreview);
+    }
+    else
+    {
+        // we hope that auto will select the best photo pin options
+        settings.PhotoCaptureSource(PhotoCaptureSource::Auto);
+    }
+
+    settings.StreamingCaptureMode(StreamingCaptureMode::Video);
+    mediaCapture.InitializeAsync(settings).get();
+
+    //Set format on the medicapture frame source

```

Comment:
>//S [](start = 4, length = 3)

nit: add a space. Also adopt either 1st letter in caps or small caps in your comments everywhere. .. sry I know I am OCD about this.